js/core/options/index.js

```diff
@@ -197,4 +199,14 @@ export class Options {
     isDeprecated(name) {
         return Object.prototype.hasOwnProperty.call(this._deprecated, name);
     }
+
+    cache(name, options) {
```

Comment:
Why not split into two simple methods? E.g. `addToCache` \ `readFromCache` or simpler `cahe` \ `fromCache`

Reply:
No need for this `if(isGetter) {` logic, simpler to read - good points only 😉

Reply:
We discussed this with @churkin and desided to make this method like `option` and `silent` methods.
Should we change them too?

Reply:
No you shouldn't
I don't see how these methods are related to each other actually. `cache` is very simple and should not be over-weighted. `addToCache` and `getFromCache` is very clear, no one is going to be mixed up, I think.

Reply:
Methods `option`, `silent` and `cache` work like getter and setter at same time. I think we should leave them as they are now or refactor them all.

Reply:
You should not merge PR if it has not resolved comments.

Reply:
Decided to leave them as they are now